hoot-core/src/main/cpp/hoot/core/ops/NamedOp.cpp

```diff
@@ -59,87 +59,106 @@ void NamedOp::apply(boost::shared_ptr<OsmMap> &map)
   Factory& f = Factory::getInstance();
 
   QElapsedTimer timer;
-  LOG_DEBUG("Applying map operations...");
   foreach (QString s, _namedOps)
   {
-    timer.restart();
     if (s.isEmpty())
     {
-      // pass
+      return;
     }
-    else if (f.hasBase<OsmMapOperation>(s.toStdString()))
+
+    timer.restart();
+    LOG_DEBUG("Element count before operation: " << map->getElementCount());
+    if (f.hasBase<OsmMapOperation>(s.toStdString()))
     {
```

Comment:
I'm sure this whole mess below can be cleaned up better with a templated function, just haven't gotten around to it yet.